frontend/src/pages/EventDashboard/Hackerpack/index.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {
+  Box,
+  Avatar,
+  Typography,
+  Button,
+  Grid,
+  Divider
+} from "@material-ui/core";
+
+import Markdown from "components/generic/Markdown";
+
+import PageHeader from "components/generic/PageHeader";
+import PageWrapper from "components/layouts/PageWrapper";
+
+import { OutboundLink } from "react-ga";
+
+import styles from "../EventDashboard.module.scss";
+
+import Partners from "constants/hackerpack-partners.js";
+
+const Hackerpack = () => {
+  const CompanySection = ({ name, icon, description, link }) => {
+    return (
+      <Grid container item direction="row" spacing={5}>
+        <Grid item xs={4}>
+          <img alt={name} src={icon} className={styles.companyLogo} />
+        </Grid>
+        <Grid item xs={8}>
+          <Typography variant="h5">{name}</Typography>
+          <Typography>
+            <Markdown source={description} />
+          </Typography>
+          <Box className={styles.outboundLink}>
+            <OutboundLink eventLabel="myLabel" to={link} target="_blank">
+              <Button color="secondary" variant="contained">
```

Comment:
Could use components/generic/Button here, it's a drop in replacement e.g. just change the import. After that change to color="theme_turquoise"

Reply:
Done